application/controllers/handler.js

```diff
@@ -49,10 +49,24 @@ module.exports = {
       const fileType = '.jpeg';
       const filePath = path.join(conf.fsPath, 'slideThumbnails/' + fileName + fileType);
       const html = request.payload;
+      let document = cheerio.load(html);
+      let pptxwidth = document('div[class=pptx2html]').css().width.replace('px', '');
+      let pptxheight = document('div[class=pptx2html]').css().height.replace('px', '');
+      pptxwidth = pptxwidth ? pptxwidth : 0;
+      pptxheight = pptxheight ? pptxheight : 0;
+      let width = 1024;
+      let height = 768;
+      if (pptxwidth !== 0 && pptxheight !== 0) {
+        width = pptxwidth;
+        height = pptxheight;
+      } else {
+        width = 'all';
```

Comment:
According to the webshot documentation, there is no option "all" for windowSize, see https://www.npmjs.com/package/webshot#options

Reply:
Allan said to me in chat that everything from slidewiki-platform will contain dimensions, thus it's not an issue and ready to merge.